scripts/qa_crowbarsetup.sh

```diff
@@ -2517,7 +2517,13 @@ function onadmin_addupdaterepo()
     if [[ -n "$UPDATEREPOS" ]]; then
         local repo
         for repo in ${UPDATEREPOS//+/ } ; do
-            wget --progress=dot:mega -r --directory-prefix $UPR --no-parent --no-clobber --accept x86_64.rpm,noarch.rpm $repo || exit 8
+            wget --progress=dot:mega \
+                -r --directory-prefix $UPR \
+                --no-parent \
+                --no-clobber \
+                --accept x86_64.rpm,noarch.rpm \
+                ${repo%/}/ \
```

Comment:
There are also complex regexes in other languages, that are hard to decipher. Just because some of us don't know all bashisms should not prevent the usage of it. Rather take it as opportinity to learn it (or work on migrating mkcloud to a different language).

My proposed fix will remove "all" trailing slashes, while @aspiers code will only remove the last one.